test/tomo/operators/ray_trafo_test.py

```diff
@@ -155,9 +172,24 @@ def test_projector(projector):
     expected_max = projector.domain.partition.extent()[0] * np.sqrt(2)
     assert almost_equal(proj.ufunc.max(), expected_max, places=places)
 
-    # Adjoint definition <Ax, Ax> = <x, A*A x>
+
+def test_adjoint(projector):
```

Comment:
Split this into two tests, which caused a 0.5s longer runtime.  I find this acceptable since it allows us to use different phantoms (specifically, a more advanced one for the adjoint test).